Realm/RLMApp.h

```diff
@@ -42,7 +42,7 @@ typedef void(^RLMOptionalErrorBlock)(NSError * _Nullable);
 @interface RLMAppConfiguration : NSObject <NSCopying>
 
 /// A custom base URL to request against.
-@property (nonatomic, strong, nullable) NSString *baseURL;
+@property (nonatomic, strong, null_resettable) NSString *baseURL;
```

Comment:
Also, the docstring should be updated to reflect that now we return the default url if there is no given base url, if we are going in that direction